grunt/concurrent.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
 	dev: {
-		tasks: ['nodemon:debug', 'concat:js', 'uglify:dev', 'watch'],
+		tasks: ['nodemon', 'concat', 'watch'],
```

Comment:
uglify is not needed on dev, and calling a taks without `:`, just executes all tasks